src/test/Recourse/NotificationUrlJsons/ScheduledPayment.json

```diff
@@ -0,0 +1,123 @@
+          ],
+          "anchor_uuid": "407b2959-3df2-4c72-8c44-c90d8f00855f"
+        },
+        "public_nick_name": "Jodi (nickname)"
+      },
+      "label_schedule_user_canceled": null,
+      "payment": {
+        "amount": {
+          "currency": "EUR",
+          "value": "-8.89"
+        },
+        "alias": {
+          "iban": "NL71BUNQ2025104162",
+          "is_light": false,
+          "display_name": "J. Barrett",
+          "avatar": {
+            "uuid": "8418f60a-2f89-443e-94ab-5f4cc10a1a1f",
+            "image": [
+              {
+                "attachment_public_uuid": "c00867c4-8bbe-4dd7-ac72-85798a2969cb",
+                "height": 1024,
+                "width": 1024,
+                "content_type": "image\/png"
+              }
+            ],
+            "anchor_uuid": null
+          },
+          "label_user": {
+            "uuid": "407b2959-3df2-4c72-8c44-c90d8f00855f",
+            "display_name": "J. Barrett",
+            "country": "NL",
+            "avatar": {
+              "uuid": "3872244f-82c5-4131-82d9-ba878f7e4cb6",
+              "image": [
+                {
+                  "attachment_public_uuid": "abca51a8-af0f-4ab5-b329-645809781615",
+                  "height": 480,
+                  "width": 480,
+                  "content_type": "image\/jpeg"
+                }
+              ],
+              "anchor_uuid": "407b2959-3df2-4c72-8c44-c90d8f00855f"
+            },
+            "public_nick_name": "Jodi (nickname)"
+          },
+          "country": "NL"
+        },
+        "counterparty_alias": {
+          "iban": "NL59BUNQ2025104669",
+          "is_light": false,
+          "display_name": "Alpha Corp.",
+          "avatar": {
+            "uuid": "26789d97-ec34-4fe8-9a71-ca145a23ba7a",
+            "image": [
+              {
+                "attachment_public_uuid": "3965b302-7a77-4813-8b00-ad3f9b84f439",
+                "height": 1024,
+                "width|": 1024,
+                "content_type": "image\/png"
+              }
+            ],
+            "anchor_uuid": null
+          },
+          "label_user": {
+            "uuid": "2711664e-885e-4b5c-bf2c-581ba39061d3",
+            "display_name": "Alpha Corp.",
+            "country": "NL",
+            "avatar": {
+              "uuid": "829fee85-ffa6-473b-ac5b-b464c7b0a3a9",
+              "image": [
+                {
+                  "attachment_public_uuid": "89347f1e-fd96-4c28-b9f5-bc9ec1f4443a",
+                  "height": 640,
+                  "width": 640,
+                  "content_type": "image\/png"
+                }
+              ],
+              "anchor_uuid": "2711664e-885e-4b5c-bf2c-581ba39061d3"
+            },
+            "public_nick_name": "Alpha Corp."
+          },
+          "country": "NL"
+        },
+        "description": "Test payment to NL59BUNQ2025104669",
+        "merchant_reference": null,
+        "type": "BUNQ",
+        "sub_type": "PAYMENT"
+      }
+    }
+  }
+}
+
```

Comment:
Some of the JSON files in java seem to have _too many_ newlines in the end. It does not matter that much, but would be cool if you could fix it for consistency anyways :)